frida_mode/qemu_parallel.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+export PWD=$(pwd)
+export PARENT=$(realpath $PWD/..)
+export TEST_BIN=$PWD/build/test/pngtest
+export TEST_DATA_DIR=$PWD/build/test/libpng-1.2.56/contrib/pngsuite/
+export QEMU_OUT=$PWD/qemu_out
+export TESTINSTR_BASE_QEMU=0x4000000000
+
+export AFL_QEMU_PERSISTENT_ADDR=$($PWD/test/get_symbol_addr.py -f $TEST_BIN -s main -b $TESTINSTR_BASE_QEMU)
+export AFL_QEMU_INST_RANGES=$($PWD/test/get_section_addrs.py -f $TEST_BIN -s .text -b $TESTINSTR_BASE_QEMU)
+export AFL_QEMU_PERSISTENT_GPR=1
+
+echo AFL_QEMU_PERSISTENT_ADDR = $AFL_QEMU_PERSISTENT_ADDR
+echo AFL_QEMU_INST_RANGES = $AFL_QEMU_INST_RANGES
+
+rm -rf $QEMU_OUT
+
+tmux new-window -n:master  "AFL_QEMU_PERSISTENT_ADDR=$AFL_QEMU_PERSISTENT_ADDR AFL_QEMU_INST_RANGES=$AFL_QEMU_INST_RANGES AFL_QEMU_PERSISTENT_GPR=1 $PARENT/afl-fuzz -V60 -Q -i $TEST_DATA_DIR -o $QEMU_OUT -D master  -- $TEST_BIN @@"
```

Comment:
3rd time.
In any case, do we want these scripts?

Reply:
These scripts are gone now.